include/pfs/drp/stella/traces.h

```diff
@@ -106,6 +107,24 @@ void centroidPeak(
 );
 
 
+/// Median filter columns of an image
+///
+/// This is intended to remove the traces, to allow the cosmic ray finder to
+/// operate on an image (more) devoid of continuum sources which can cause
+/// false positive CRs.
+///
+/// @param image : Image to which to apply the median filter
+/// @param mask : Pixels that should be ignored (true means a bad value)
+/// @param halfHeight : Half-height of the median filter (pixels)
+/// @returns Median filter image
+template <typename T>
+ndarray::Array<T, 2, 1> medianFilterColumns(
```

Comment:
Is this code actually used in your CR detection algorithm? I couldn't see the call.

Reply:
Right [here](https://github.com/Subaru-PFS/drp_stella/blob/tickets/PIPE2D-1063/python/pfs/drp/stella/repair.py#L95).